.circleci/create_circleci_config.py

```diff
@@ -495,6 +494,8 @@ def job_name(self):
     pipelines_tf_job,
 ]
 REPO_UTIL_TESTS = [repo_utils_job]
+DOC_TESTS = [doc_test_job]
```

Comment:
We need to separate this job from `REGULAR_TESTS`, as the trigger condition is different. Here the condition is to run when the `doctest_list.txt` is not empty (i.e. diff contains some doc examples)